### DISCUSSION TITLE
Redirecting alternative URI schemes (`mailto:`, `tel:`, etc)

### DISCUSSION THREAD

**Discussion by axelerometer** [einaregilsson/Redirector#406] [Q&A]
I would find it useful to redirect alternative URI schemes: `mailto:` and `tel:` URIs, redirecting them to a web URL. Seemingly this is not supported by Redirector.

I can create a redirect rule with a pattern such as `^tel:(.+)$`, it applies correctly in the "Example result" preview, but actual URLs on sites don't seem to be handled.

Any ideas/suggestions? I am using Firefox.

**Comment by Gitoffthelawn** [comment]
I think it would be worthwhile too.  The big question is if that's supported in Web Extensions.  IIRC, last I looked (which was a long time ago), that functionality wasn't supported by Web Extensions.  But Web Extension technology is constantly improving, so maybe it's possible now.  Does anyone know?
